MdeModulePkg/Library/BaseIpmiCommandLibNull/IpmiCommandLibNetFnSensor.c

```diff
@@ -0,0 +1,48 @@
+/** @file
+  IPMI Command - NetFnApp NULL instance library.

```

Comment:
It looks like this may have come from MdeModulePkg\Library\BaseIpmiCommandLibNull\IpmiCommandLibNetFnApp.c.

 ```suggestion
  IPMI Command - NetFnSensor NULL instance library.

```

Reply:
Changed to NetFnSensor